packages/ui/src/styles/globals.css

```
@@ -6,6 +6,7 @@
 @custom-variant dark (&:is(.dark *));
 
 @source "../../../apps/**/*.{ts,tsx}";
+@source "../../../components/**/*.{ts,tsx}";
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)